Packs/MajorBreachesInvestigationandResponse/ReleaseNotes/1_3_1.md

```diff
@@ -0,0 +1,10 @@
+
```

Comment:
This playbook includes the following tasks:
 - Collect indicators to be used in your threat hunting process
 - Retrieve IOCs related to HAFNIUM  and the exploited exchange 0-day vulnerabilities
 - Discover IOCs related to the attack 
 - Query firewall logs to detect malicious network activity 
 - Search endpoint logs for malicious hashes to detect compromised hosts (Available from Cortex XSOAR 5.5.0).
 - Block indicators